docs/source/tutorials/workflows.rst

```diff
@@ -9,13 +9,20 @@ Please note that we are rolling this feature out into open availability in `Beta
 in development and will be added progressively. 
 
 1. Creating a `Workflow` project
-====================================
+===================================
+
+Workflow projects can either be created from scratch, or based on a template. 
+
+1. At the moment, `Workflow` projects have to be created from scratch using the Encord web-app. Please see our tutorial on how to `create Workflow projects <https://docs.encord.com/projects/workflows/creating-and-configuring-workflows/>`_ to learn how this is done.
```

Comment:
That doesn't make much sense to me. The whole section should essentially say something like "you can create workflow projects via choosing a template id. You can only create new templates via the UI and need to get the id from there".